packages/playground/blueprints/src/lib/steps/run-sql-query.ts

```diff
@@ -0,0 +1,50 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlQuery
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlQuery",
+ * 		"path": "wordpress.org/plugins"
+ * }
+ * </code>
+ */
+export interface RunSqlQueryStep {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSqlQuery';
+	/**
+	 * The SQL query to run.
```

Comment:
The description here disagrees with the type and is a bit confusing. Maybe `The SQL query or array of queries to run.`

Reply:
@dmsnell  This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files